src/components/Contacts/MassActions/AddTags/MassActionsAddTagsModal.tsx

```diff
@@ -70,6 +70,38 @@ export const MassActionsAddTagsModal: React.FC<
     },
   });
 
+  const handleValidation = async (fields: Partial<ContactUpdateInput>) => {
+    const contactsExistingTags =
+      contactsForTags?.contacts.nodes.map((contact) => ({
+        tagList: [...new Set([...contact.tagList])],
+      })) ?? [];
+
+    const existingTags = [];
+    contactsExistingTags.forEach((contact) => {
+      for (let i = 0; i < contact.tagList.length; i++) {
+        existingTags.push(contact.tagList[i]);
+      }
+    });
+
+    for (let i = 0; i < fields.tagList?.length; i++) {
+      existingTags.push(fields.tagList[i]);
```

Comment:
It looks like you have some typescript errors. You probably need to add ? optional chaining.

Reply:
@cjpastika can you push the changes we've made to fix the TS errors